stores/Todo.store.ts

```diff
@@ -56,6 +57,12 @@ export class TodoStore {
     this.saveTodos();
   };
 
+  editChecked = (todo: Todo, checked: boolean) => {
```

Comment:
This is a recommendation for better decoupling. Not that your example does not work.

Instead of passing the the whole Todo object in this form, I would pass just an ID of the Todo task (or the index).

So the 'internals' of how we toggle a todo is purely to the store.
And have this function check the list and update accordingly, etc. 

Again, not that your implementation does not work.

Reply:
All the code that calls this already has access to a `Todo`, so passing an ID would be inefficient and error prone. I don't see how this would help with decoupling either, since the UI layer is simply passing the object to the store layer that needs mutation.